ospfd/ospf_opaque.c

```diff
@@ -560,6 +560,9 @@ register_opaque_info_per_type (struct ospf_opaque_functab *functab,
       listnode_add (new->area->opaque_lsa_self, oipt);
       break;
     case OSPF_OPAQUE_AS_LSA:
+      if (IS_DEBUG_OSPF_TRACE)
```

Comment:
this opaue lsa retrieves ospf instace differently then others. The trace is meant for internal daemon activity/probing.  I have addressed in latest patch.